.github/workflows/ci.yml

```
@@ -115,29 +127,27 @@ jobs:
     steps:
     - uses: actions/checkout@v4
-    - name: Set up date and time variable
-      run: echo "DATE_TIME=$(TZ='Asia/Tokyo' date '+%Y-%m-%d-%H-%M-%S')" >> $GITHUB_ENV
     - name: Output date and time
       run: |
         ls -all;pwd;whoami;
-        echo "Date and Time: ${{ env.DATE_TIME }}"
+        echo "Date and Time: ${{ env.DATETIME }}"
     - name: Test build
       run: |
         #docker login --username ${{ github.actor }} --password ${{ secrets.GITHUB_PAT }} ghcr.io 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)